tests/pages/pages.py

```diff
@@ -627,7 +627,7 @@ class EditEmailTemplatePage(BasePage):
     def folder_path_item(folder_name):
         return (
             By.XPATH,
-            "//a[contains(@class,'folder-heading-folder') and contains(text(), '{}')]".format(
+            "//a[contains(@class,'folder-heading-folder')]/text()[contains(.,'{}')]".format(
```

Comment:
This xpath returns the text node but the old one returned the `a` element - do you know if this matters?

I think if we want to still have it return the `a` we can change it to:

```suggestion
            "//a[contains(@class,'folder-heading-folder')]/text()[contains(.,'{}')]/..".format(
```

<img width="689" alt="image" src="https://user-images.githubusercontent.com/2920760/234269093-7c7a5dd5-2eff-4dbf-9e6a-490289c1ff63.png">